frontend/packages/console-app/src/components/nodes/NodesPage.tsx

```diff
@@ -359,7 +359,7 @@ const NodesTableRow: React.FC<RowProps<NodeKind, GetNodeStatusExtensions>> = ({
         id={nodeColumnInfo.created.id}
         activeColumnIDs={activeColumnIDs}
       >
-        <Timestamp timestamp={node.metadata.creationTimestamp} />
+        <Timestamp timestamp={node.metadata?.creationTimestamp || ''} />
```

Comment:
should no longer be needed as Timestamp now accepts null

```suggestion
        <Timestamp timestamp={node.metadata?.creationTimestamp} />
```